test/integration/api/index.js

```diff
@@ -478,6 +485,51 @@ describe('basicAPIs', () => {
     });
   });
 
+  describe('Errors handling', () => {
```

Comment:
I think it's better to have unit tests with mocks for that, isn't it?

Reply:
unit tests also have been added